### PR TITLE
Add landingpage

### PR DESCRIPTION
**Pull request by FaGru**

### CODE REVIEW

File: src/GlobalStyle.js [hunk 1]
```diff
@@ -42,6 +43,7 @@ const GlobalStyle = createGlobalStyle`
       'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
       sans-serif;
     font-size: 1rem;
+
```
Comment (JelenadlS) [inline src/GlobalStyle.js:46]: Have look to the empty rows, line 3 is too much I think, 46 as well and between 47 and 48 😊

File: src/pages/LandingPage.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
+import styled from 'styled-components';
+import { useNavigate } from 'react-router-dom';
+import useStore from '../hooks/useStore';
+import backround from '../images/background.jpg'
+
+export default function LandingPage() {
+  let navigate = useNavigate();
+  const setInstructionOneVisible = useStore(
+    state => state.setInstructionOneVisible
```
Comment (JelenadlS) [inline src/pages/LandingPage.js:9]: also here, or is it because of useStore - then maybe you could do it there once?

**Review by JelenadlS — APPROVED** [review]
Just some small things i would say😊

File: src/components/InstructionsSequencer.js [hunk 3]
```diff
@@ -16,6 +16,9 @@ export default function InstructionsSequencer() {
   const setInstructionFiveVisible = useStore(
     state => state.setInstructionFiveVisible
   );
+  const setIsInstructionNavVisible = useStore(
+    state => state.setIsInstructionNavVisible
+  );
```
[the remark this answers is not in the document]
Comment (FaGru) [reply src/components/InstructionsSequencer.js:21]: I could also do it in the app, but then i have to pass it through each component. I import it directly in the component with useStore

**Review by ttaannjjaa — APPROVED** [review]
looks very good 😊

**Pull request merged**